app/src/main/res/values/strings.xml

```diff
@@ -301,13 +301,13 @@
     <string name="advanced_unlock_explanation_summary">Use advanced unlocking to open a database more easily</string>
     <string name="biometric_unlock_enable_title">Biometric unlocking</string>
     <string name="biometric_unlock_enable_summary">Lets you scan your biometric to open the database</string>
-    <string name="biometric_auto_open_prompt_title">Auto-open biometric prompt</string>
-    <string name="biometric_auto_open_prompt_summary">Automatically ask for biometric if the database is set up to use it</string>
+    <string name="biometric_auto_open_prompt_title">Auto-open biometric scan</string>
```

Comment:
It depends on the functioning of the device, the scan can be activated afterwards. Only the API view which prompts to start the scanning procedure is displayed.

Reply:
Isn't that the scan? Like the scan is triggered from the API. I think it would be easier to understand what it does if it said "scan". I understand it isn't as technically correct.

Reply:
As we use an API, there is no guarantee that it will scan. There may be an error and a wait request indicated in the prompt. I just know there will be a prompt related to bio-metrics, not what it asks for.

Reply:
Well the intent is still to get to a scan?
Having to wait, or seeing an error is not the process the user aligns to, as they are out of the ordinary, or different from the action carried out.